starter-code/css/main.css

```diff
@@ -0,0 +1,88 @@
+
+* {
+  font-family: Lato;
+  font-weight: 300;
+}
+
+body {
+  margin: 75px 50px;
+}
+
+h1 a {
+  font-size: 36pt;
+  text-decoration: none;
+  color: #f8cd84;
+}
+
+header nav a {
+  color: #e5e6ed;
+  text-decoration: none;
+  border-bottom: 1px solid;
+  padding-bottom: 3px;
+  margin: 0 7px;
+  font-size: 15pt;
+}
+
+header nav a:hover, header nav a:focus {
+  color: #f8cd84;
+}
+
+article {
+  max-width: 650px;
+  border-bottom: 1px solid;
+  border-color: #e5e6ed;
+  margin: 70px 0;
+}
+
+article h3 {
+  font-size: 24pt;
+  color: #e5e6ed;
+  margin-bottom: 0;
+}
+
+h2 a {
+  color: black;
+  text-decoration: none;
+  font-size: 24pt;
+}
+
+h4 {
+  text-align: right;
+}
+
+h4 a {
+  text-decoration: none;
+  font-weight: 700;
+  color: #f8cd84;
+}
+
+article p::first-letter {
+  color: #e5e6ed;
+  font-size: 72pt;
+  float: left;
+  line-height: 66%;
+  padding: 5px 10px 10px 10px;
```

Comment:
shorthand way to write this: `padding: 5px 10px 10px;` which will be top, left + right, bottom.